tests/estimators/object_detection/test_pytorch_yolo.py

```diff
@@ -173,116 +173,116 @@ def test_loss_gradient(art_warning, get_pytorch_yolo):
-                1.70234252e-05,
-                -1.04990377e-05,
-                -2.20760157e-05,
-                2.89993841e-05,
-                2.43352024e-05,
-                5.40295805e-05,
-                -3.54345357e-05,
-                1.49476218e-05,
-                -1.83201955e-05,
-                -4.52892618e-06,
-                -2.10271646e-05,
-                -1.38741379e-05,
-                1.19676406e-06,
-                1.44154410e-05,
-                -1.16514025e-06,
-                -2.48137949e-05,
-                -4.98828103e-06,
-                1.53667770e-05,
-                3.29377417e-06,
-                2.14245338e-05,
-                1.82093459e-06,
-                2.11810093e-05,
-                1.19740625e-05,
-                1.71546981e-06,
-                -1.24681810e-05,
-                -7.98507535e-05,
-                -3.12026459e-05,
-                1.01383330e-05,
-                -3.08082854e-05,
-                -2.56484091e-05,
-                -6.88045111e-05,
-                1.62506112e-05,
-                -1.15138228e-05,
-                -9.07951107e-06,
-                1.25368360e-05,
-                -1.09746734e-05,
+                0.012576922,
+                -0.005133151,
+                -0.0028872574,
+                -0.0029357928,
+                -0.008929219,
+                0.012767567,
+                -0.00715934,
+                0.00987368,
+                -0.0014089097,
+                -0.004765472,
+                -0.007845592,
+                -0.0065127434,
+                -0.00047654763,
+                -0.018194549,
+                0.00025652442,
+                -0.01420591,
+                0.03873131,
+                0.080963746,
+                -0.009225381,
+                0.026824722,
+                0.005942673,
+                -0.025760904,
+                0.008754236,
+                -0.037260942,
+                0.027838552,
+                0.0485742,
+                0.020763855,
+                -0.013568859,
+                -0.0071423287,
+                0.000802512,
+                0.012983642,
+                0.006466129,
+                0.0025194373,
+                -0.012298459,
+                -0.01168492,
+                -0.0013298508,
+                -0.007176587,
+                0.01996972,
+                -0.004173076,
+                0.029163878,
+                0.022482246,
+                0.008151911,
+                0.025543496,
+                0.0007374112,
+                0.0008220682,
+                -0.005740379,
+                0.009537468,
+                -0.01116704,
+                0.0010225883,
+                0.00026052812,
             ]
         )
 
-        np.testing.assert_array_almost_equal(grads[0, 0, 208, 175:225], expected_gradients1, decimal=2)
+        np.testing.assert_array_almost_equal(grads[0, 0, 208, 175:225], expected_gradients1, decimal=1)
```

Comment:
Is the precision change required?

Reply:
Due to normalizing the inputs, the gradients have now changed in magnitude and are much larger now. Therefore, the decimal precision needs to be decreased.